### PR TITLE
fix(Ethereum_SpokePool): Correct admin/owner comment

### PR DESCRIPTION
**Pull request by pxrl**

### CODE REVIEW

File: contracts/Ethereum_SpokePool.sol [hunk 1]
```diff
@@ -32,7 +32,7 @@ contract Ethereum_SpokePool is SpokePool, Ownable {
         IERC20(relayerRefundLeaf.l2TokenAddress).safeTransfer(hubPool, relayerRefundLeaf.amountToReturn);
     }
 
-    // Admin is simply owner which should be same account that owns the HubPool deployed on this network. A core
-    // assumption of this contract system is that the HubPool is deployed on Ethereum.
+    // The SpokePool deployed to the same network as the HubPool must be owned by the HubPool.
+    // A core assumption of this contract system is that the HubPool is deployed on Ethereum.
```
Comment (nicholaspai) [inline contracts/Ethereum_SpokePool.sol:36]: I think this is a good place to mention that `crossDomainAdmin` is also unused for this spoke pool and arbitrarily set to the deployer. There is no harm leaving it as the deployer

**Review by nicholaspai — APPROVED** [review]
Great suggestion! I have one additional comment I'd like to see

File: contracts/Ethereum_SpokePool.sol (hunk 1 above)
Comment (pxrl) [reply contracts/Ethereum_SpokePool.sol:36]: > I think this is a good place to mention that `crossDomainAdmin` is also unused for this spoke pool and arbitrarily set to the deployer. There is no harm leaving it as the deployer

Any thoughts on [this](https://github.com/across-protocol/contracts-v2/pull/217/commits/3be288a773d1106376d9343669423b1b6c23a971) instead?

Comment (nicholaspai) [reply contracts/Ethereum_SpokePool.sol:36]: yea thats more consistent but still add a comment why we're setting `crossDomainAdmin` to that

**Review by nicholaspai — APPROVED** [review]

File: contracts/Ethereum_SpokePool.sol [hunk 2]
```diff
@@ -14,6 +14,7 @@ contract Ethereum_SpokePool is SpokePool, Ownable {
 
     /**
      * @notice Construct the Ethereum SpokePool.
+     * @dev crossDomainAdmin is unused on Ethereum.
```
Comment (nicholaspai) [inline contracts/Ethereum_SpokePool.sol:17]:
```suggestion
     * @dev crossDomainAdmin is unused on this contract.
```

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**